src/V3FactoryOwner.sol

```diff
@@ -177,7 +177,9 @@ contract V3FactoryOwner {
   /// method.
   /// @return _amount0 The amount0 fees collected, returned by the pool's collectProtocol method.
   /// @return _amount1 The amount1 fees collected, returned by the pool's collectProtocol method.
-  /// @dev See docs on IUniswapV3PoolOwnerActions for more information on forwarded params.
+  /// @dev The `UniswapV3Pool` contract only allows claiming the total amount minus 1 wei or less
+  /// for each token. We highly recommend checking the documentation on `IUniswapV3PoolOwnerActions`
```

Comment:
I don't know if we recommend the interface documentation -- is it this one? https://docs.uniswap.org/contracts/v3/reference/core/interfaces/pool/IUniswapV3PoolOwnerActions -- if so, it's not great, and makes no reference the issue at hand.

Reply:
Should we direct people to the `UniswapV3Pool` contract instead? The idea with the second sentence is to encourage people to look at the underlying contract before calling the method.

Reply:
yeah that might be better, as the weirdness is only expressed in the contract code itself afaik

Reply:
https://github.com/uniswapfoundation/UniStaker/pull/73/commits/2b0cf560a5f07178f9b7fee6443ac68b0232c391